docs/installation/advanced/integrating-from-source-vite.md

```diff
@@ -0,0 +1,266 @@
+
+You can install packages using npm like `npm install @ckeditor/ckeditor5-editor-classic`, or copy dependencies from the build repo and typing `npm install`. The `dependencies` section of your `package.json` should look similar to this:
+
+```js
+"dependencies": {
+	// Dependencies.
+	// ...
+
+    "@ckeditor/ckeditor5-adapter-ckfinder": "^x.y.z",
+    "@ckeditor/ckeditor5-autoformat": "^x.y.z",
+    "@ckeditor/ckeditor5-basic-styles": "^x.y.z",
+    "@ckeditor/ckeditor5-block-quote": "^x.y.z",
+    "@ckeditor/ckeditor5-easy-image": "^x.y.z",
+    "@ckeditor/ckeditor5-editor-classic": "^x.y.z",
+    "@ckeditor/ckeditor5-essentials": "^x.y.z",
+    "@ckeditor/ckeditor5-heading": "^x.y.z",
+    "@ckeditor/ckeditor5-image": "^x.y.z",
+    "@ckeditor/ckeditor5-link": "^x.y.z",
+    "@ckeditor/ckeditor5-list": "^x.y.z",
+    "@ckeditor/ckeditor5-paragraph": "^x.y.z",
+    "@ckeditor/ckeditor5-theme-lark": "^x.y.z",
+    "@ckeditor/ckeditor5-upload": "^x.y.z"
+
+    // More dependencies.
+	// ...
+}
+```
+
+## Vite configuration
+
+When your editor has all the necessary plugins, you can move on to integration with Vite. There is also an official plugin for this purpose. It handles loading the SVG icons and styles from the packages and the theme package. You can install it via the below command.
+
+```js
+npm install --save @ckeditor/ckeditor5-vite-plugin
+```
+
+The plugin is installed but will not work yet, so you need to add it to the Vite configuration. First, add the `vite.config.js` file at the root of your project (or use an existing one). Then, modify the file by adding the following lines of code.
+
+```js
+// vite.config.js
+
+import { defineConfig } from 'vite';
+import ckeditor5 from '@ckeditor/vite-plugin-ckeditor5';
+
+export default defineConfig( {
+	plugins: [
+		ckeditor5( { theme: require.resolve( '@ckeditor/ckeditor5-theme-lark' ) } )
+	]
+} );
+```
+
+The configuration slightly differs for ESM projects. If you try to start the dev server using the `npm run dev` command, you may encounter an error: `require.resolve is not a function`. In this case, you need some additional lines of code.
+
+```js
+// vite.config.js
+
+import { createRequire } from 'node:module';
+const require = createRequire( import.meta.url );
+```
+
+Now your setup is complete, and you can start using the editor.
+
+## Running the editor – method 1
+
+You can now import all the needed plugins and the creator directly into your code and use it there. The easiest way to do so is to copy it from the `src/ckeditor.js` file available in every build repository.
```

Comment:
Where to paste it in the scaffolded project?

Reply:
This section is from the webpack integrating guide. Should I change it in both places?

Reply:
Webpack guide doesn't use any scaffolding so the place is not defined. Here I think we should write something like this: If you scaffolded the project using a vite template, paste the following code into main.js.